xbmc/addons/kodi-addon-dev-kit/doxygen/Modules/modules_python.dox

```diff
@@ -141,3 +141,27 @@ web applications or frameworks for the Python programming language.
 @}
 
 */
+/** @page python_v14 Python v14 API Changes
+ *  @brief Overview of changes done in Kodi v14 Python API
+ *
+ *  Changes page contents.
+ */
+
```

Comment:
Here about a global to have all of them in one another Page together and not on top:

/** @page python_revisions Python API Changes
  *  @brief Overview of changes done Python API for Kodi
  *
  *  @subpage python_v14 
  *  @subpage python_v15
  *  @subpage python_v16
  *  @subpage python_v17
  */